lib/web_ui/lib/src/ui/window.dart

```diff
@@ -241,173 +241,91 @@ class Locale {
-      case 'meg':
-        return 'cir'; // Mea; deprecated 2013-09-10
-      case 'mst':
-        return 'mry'; // Cataelano Mandaya; deprecated 2010-03-11
-      case 'mwj':
-        return 'vaj'; // Maligo; deprecated 2015-02-12
-      case 'myt':
-        return 'mry'; // Sangab Mandaya; deprecated 2010-03-11
-      case 'nad':
-        return 'xny'; // Nijadali; deprecated 2016-05-30
-      case 'ncp':
-        return 'kdz'; // Ndaktup; deprecated 2018-03-08
-      case 'nnx':
-        return 'ngv'; // Ngong; deprecated 2015-02-12
-      case 'nts':
-        return 'pij'; // Natagaimas; deprecated 2016-05-30
-      case 'oun':
-        return 'vaj'; // !O!ung; deprecated 2015-02-12
-      case 'pcr':
-        return 'adx'; // Panang; deprecated 2013-09-10
-      case 'pmc':
-        return 'huw'; // Palumata; deprecated 2016-05-30
-      case 'pmu':
-        return 'phr'; // Mirpur Panjabi; deprecated 2015-02-12
-      case 'ppa':
-        return 'bfy'; // Pao; deprecated 2016-05-30
-      case 'ppr':
-        return 'lcq'; // Piru; deprecated 2013-09-10
-      case 'pry':
-        return 'prt'; // Pray 3; deprecated 2016-05-30
-      case 'puz':
-        return 'pub'; // Purum Naga; deprecated 2014-02-28
-      case 'sca':
-        return 'hle'; // Sansu; deprecated 2012-08-12
-      case 'skk':
-        return 'oyb'; // Sok; deprecated 2017-02-23
-      case 'tdu':
-        return 'dtp'; // Tempasuk Dusun; deprecated 2016-05-30
-      case 'thc':
-        return 'tpo'; // Tai Hang Tong; deprecated 2016-05-30
-      case 'thx':
-        return 'oyb'; // The; deprecated 2015-02-12
-      case 'tie':
-        return 'ras'; // Tingal; deprecated 2011-08-16
-      case 'tkk':
-        return 'twm'; // Takpa; deprecated 2011-08-16
-      case 'tlw':
-        return 'weo'; // South Wemale; deprecated 2012-08-12
-      case 'tmp':
-        return 'tyj'; // Tai Mène; deprecated 2016-05-30
-      case 'tne':
-        return 'kak'; // Tinoc Kallahan; deprecated 2016-05-30
-      case 'tnf':
-        return 'prs'; // Tangshewi; deprecated 2010-03-11
-      case 'tsf':
-        return 'taj'; // Southwestern Tamang; deprecated 2015-02-12
-      case 'uok':
-        return 'ema'; // Uokha; deprecated 2015-02-12
-      case 'xba':
-        return 'cax'; // Kamba (Brazil); deprecated 2016-05-30
-      case 'xia':
-        return 'acn'; // Xiandao; deprecated 2013-09-10
-      case 'xkh':
-        return 'waw'; // Karahawyana; deprecated 2016-05-30
-      case 'xsj':
-        return 'suj'; // Subi; deprecated 2015-02-12
-      case 'ybd':
-        return 'rki'; // Yangbye; deprecated 2012-08-12
-      case 'yma':
-        return 'lrr'; // Yamphe; deprecated 2012-08-12
-      case 'ymt':
-        return 'mtm'; // Mator-Taygi-Karagas; deprecated 2015-02-12
-      case 'yos':
-        return 'zom'; // Yos; deprecated 2013-09-10
-      case 'yuu':
-        return 'yug'; // Yugh; deprecated 2014-02-28
-      default:
-        return languageCode;
```

Comment:
The behavior here looks like it might be different. Is it intentional? Does it matter?

Reply:
What's the difference in the behavior? If I'm reading the code correctly, both versions will attempt to map a deprecated value to the new value, and default to `_languageCode` otherwise.

Reply:
Ah, you're right. I missed the `?? _languageCode` on the getter.